binarySearch/binarySearch.js

```diff
@@ -11,5 +11,30 @@
 **/
 
 const binarySearch = (nums, target) => {
+    let min = 0;
+	let max = nums.length - 1;
```

Comment:
Would you like to do some work with me on them tomorrow at some point? I'm working in the morning tomorrow and off for the weekend until next Tuesday.